.github/workflows/ci.yml

```diff
@@ -60,18 +60,9 @@ jobs:
         with:
           node-version-file: .node-version
       - name: 📦 Install Dependencies
-        run: yarn install --immutable --immutable-cache
+        run: yarn install --immutable
       - name: 💅 Prettier Check
         run: yarn prettier:check
-  yarn-check-cache:
```

Comment:
The docs make it seem like `--check-cache` can still be useful to make sure that the packages in the registry match the checksums in the lockfile.

Reply:
From what I read, it only checks packages that are stored in the cache folder. Since we don't have any packages in the cache folder, it doesn't do anything.

Reply:
On second thought, I think you're right. If an attacker commits a `yarn.lock` with an incorrect checksum, it will fail to install, even without `--check-cache`.